client/src/components/Designer/Designer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Designer.css'
+
+class Designer extends React.Component {
+    
+    render() {
+        
+       return (
+          <div className="card-container">
+            <div className="pic">
+
+             </div>  
+            <div className="text">
+                <p></p>
+                <p></p>
+            </div>
+          </div>
+       )
```

Comment:
Can you populate this? I think you'll need to accept some props here to fill this stuff in